app/routes/erddap/buoy.js

```diff
@@ -1,9 +1,10 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { getMultiBuoyGeoJsonData, getBuoysCoordinates } = require('@/clients/erddap');
-const utils = require('@/utils');
-const buoys = require('@/routes/erddap/utils');
-const { cacheMiddleware } = require('@/middleware/cache');
+const utils = require("@/utils");
+const common = require("@/routes/erddap/common");
+const { cacheMiddleware } = require("@/middleware/cache");
+
+const DATASET_ID = "combined_e784_bee5_492e";
```

Comment:
I think I would called the variable `COMBINED_DATASET_ID` or something like that just to make it clear that it is the combined